docs/components/mainPage/contributors/utils/fetchContributors.ts

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+const contributorsUrl =
+  "https://api.github.com/repos/coingaming/moon-light/contributors";
+
+async function fetchContributors() {
+  if (!contributorsUrl) {
+    throw new Error("GITHUB_CONTRIBUTORS_URL is not defined");
+  }
+  try {
+    const response = await axios.get(contributorsUrl);
```

Comment:
Could we use plain fetch to avoid additional dependencies in the library?
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API